README.md

```diff
@@ -24,25 +24,27 @@
-| POSTGRES_USER `REQUIRED`        | `null`   | `postgres`                      | Username that will be used in postgres database                                                                                          |
-| POSTGRES_PASSWORD `REQUIRED`    | `null`   | `postgres`                      | Password that will be used in postgres database                                                                                          |
-| POSTGRES_IP `REQUIRED`          | `null`   | `jellystat-db` or `192.168.0.5` | Hostname/IP of postgres instance                                                                                                         |
-| POSTGRES_PORT `REQUIRED`        | `null`   | `5432`                          | Port Postgres is running on                                                                                                              |
-| JWT_SECRET `REQUIRED`           | `null`   | `my-secret-jwt-key`             | JWT Key to be used to encrypt JWT tokens for authentication                                                                              |
-| JS_BASE_URL                     | `/`      | `/`                             | Base url                                                                                                                                 |
-| JS_USER                         | `null`   | `User`                          | Master Override User in case username or password used during setup is forgotten (Both `JS_USER` and `JS_PASSWORD` required to work)     |
-| JS_PASSWORD                     | `null`   | `Password`                      | Master Override Password in case username or password used during setup is forgotten (Both `JS_USER` and `JS_PASSWORD` required to work) |
-| POSTGRES_DB                     | `jfstat` | `jfstat`                        | Name of postgres database                                                                                                                |
-| REJECT_SELF_SIGNED_CERTIFICATES | `true`   | `false`                         | Allow or deny self signed SSL certificates                                                                                               |
-| JS_GEOLITE_ACCOUNT_ID           | `null`   | `123456`                        | maxmind.com user id to be used for Geolocating IP Addresses (Can be found at https://www.maxmind.com/en/accounts/current/edit)           |
-| JS_GEOLITE_LICENSE_KEY          | `null`   | `ASDWdaSdawe2sd186`             | License key you need to generate on maxmind to use their services                                                                        |
+| Env                                 | Default  | Example                         | Description                                                                                                                              |
```

Comment:
I updated the table to be formatted consistently, made it hard to see the changes, sorry.